utilities/upgrade-service/install-upgrade.sh

```diff
@@ -1,6 +1,34 @@
 #!/usr/bin/env bash
 
-GOBIN=$(go env GOPATH)/bin
+SCRIPT_DIR="$(
+  cd "$(dirname "$0")" >/dev/null
+  pwd
+)"
+
+# deletes the temp directory
+function cleanup() {
+  rm -rf "${TEMP_DIR}"
+  echo "Deleted temp working directory $WORK_DIR"
+}
+
+# installs the stable version kustomize binary if not found in PATH
+function install_kustomize() {
+  if [[ -z "${KUSTOMIZE}" ]]; then
+    wget https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2Fv4.5.7/kustomize_v4.5.7_$(uname | tr '[:upper:]' '[:lower:]')_$(uname -m |sed s/aarch64/arm64/ | sed s/x86_64/amd64/).tar.gz -O ${TEMP_DIR}/kustomize.tar.gz
+    tar zxvf ${TEMP_DIR}/kustomize.tar.gz -C ${TEMP_DIR}
+    KUSTOMIZE=${TEMP_DIR}/kustomize
+    chmod +x ${TEMP_DIR}/kustomize
+  fi
+}
+
+# installs the stable version of kubectl binary if not found in PATH
+function install_kubectl() {
+  if [[ -z "${KUBECTL}" ]]; then
+    wget https://dl.k8s.io/release/v1.26.0/bin/$(uname | tr '[:upper:]' '[:lower:]')/$(uname -m | sed s/aarch64/arm64/ | sed s/x86_64/amd64/)/kubectl -O ${TEMP_DIR}/kubectl
+    KUBECTL=${TEMP_DIR}/kubectl
+    chmod +x ${TEMP_DIR}/kubectl
+  fi
+}
 
 # Checks if a binary is present on the local system
 exit_if_binary_not_installed() {
```

Comment:
Remove this as it is not being used

Reply:
Using this `${KUBECTL}` instead of `kubectl` that way if kubectl is not installed on the system, the one that gets installed would be used.

Reply:
What I am referring to is removing the function `exit_if_binary_not_installed` as we are not using it anymore.